pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -454,6 +454,12 @@ let
       BLK_DEV_THROTTLING        = yes;
       CFQ_GROUP_IOSCHED         = whenOlder "5.0" yes; # Removed in 5.0-RC1
       CGROUP_PIDS               = whenAtLeast "4.3" yes;
+
+      ASHMEM =                 { optional = true; tristate = whenAtLeast "5.0" "y";};
+      ANDROID =                { optional = true; tristate = whenAtLeast "5.0" "y";};
+      ANDROID_BINDER_IPC =     { optional = true; tristate = whenAtLeast "5.0" "y";};
+      ANDROID_BINDERFS =       { optional = true; tristate = whenAtLeast "5.0" "y";};
```

Comment:
Is it possible to build them as module?

Reply:
does it make sense to add nixos test here?

Reply:
Not anymore I think. That broke with 5.7.

Although, I'm not sure whether the modules were available in 5.0 yet.

Reply:
@spinus this PR already adds a NixOS test

Reply:
@Mic92 Zen kernel has patches that enable building as module.
https://github.com/zen-kernel/zen-kernel/commit/a628884e152edd7dd987eb989ddc107a2e8af713
https://github.com/zen-kernel/zen-kernel/commit/eda591e5a88289287362cadefda66f68c580800f

Reply:
I asked the zen-kernel maintainers about upstreaming this. I think we should adopt the patch for now - Many people won't need those modules by default.